cmd/backup-manager/app/backup/backup.go

```diff
@@ -85,8 +87,8 @@ func (bo *BackupOpts) dumpTidbClusterData() (string, error) {
 		fmt.Sprintf("--outputdir=%s", bfPath),
 		fmt.Sprintf("--host=%s", bo.TidbSvc),
 		"--port=4000",
-		fmt.Sprintf("--User=%s", bo.User),
-		fmt.Sprintf("--Password=%s", bo.Password),
+		fmt.Sprintf("--user=%s", bo.User),
+		fmt.Sprintf("--password=%s", bo.Password),
 		"--long-query-guard=3600",
 		"--tidb-force-priority=LOW_PRIORITY",
 		"--verbose=3",
```

Comment:
We need to exclude `mysql` database. refer:
https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-cluster/templates/scripts/_start_scheduled_backup.sh.tpl#L34